BLE_SupportedFeatures/readme.md

```diff
@@ -0,0 +1,19 @@
+# BLE Supported Features
+
+This application will list optional BLE features are supported by your device. Knowing which optional features
```

Comment:
```suggestion
This application will list BLE features supported by your device. Knowing which features
```

Not a strong opinion but I would remove _optional_ from the wording and explain that some use cases won't be possible without specific features. 

The output of this program can also be used to disable explicitly features not supported by the controller being used. Maybe we can generate bits of the json configuration from it.

Reply:
It is clear that the list generated by the code is readable. I meant if newcomers try to read the code to understand what is happening behind the scenes then putting those into a function that relies on 2 ternary operators and object oriented code is not necessarily better than just plain inline printf(). And maybe even runs slower because of the OOP overhead. I am not against it, i just do not see the point for OOP in this case.

Reply:
@pan- 
By _optional_ i meant that these BLE features are not compulsory to be implemented by the HW vendor to qualify as a BLE 5.0 hardware for example. As you say there might be BLE use cases which can only be implemented with some of these features... so yeah... the optional word can become ambiguous in these sentences.

Reply:
`The output of this program can also be used to disable explicitly features not supported by the controller being used. Maybe we can generate bits of the json configuration from it.`

Ok, if there is potential for that than OOP style gains more sense.